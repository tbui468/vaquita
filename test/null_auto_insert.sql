create database sol;
open sol;

create table planets (name string, moons int, gravity float);

insert into planets (name, gravity) values ("Mars", 2.42), ("Earth", 24.23);
select * from planets;

close sol;
drop database sol;
exit;
