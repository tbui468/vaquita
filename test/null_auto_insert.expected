created database sol
opened database sol
created table planets
inserted 2 record(s) into planets
1, Mars, null, 2.420000
2, Earth, null, 24.230000
closed database sol
dropped database sol
